### PR TITLE
Add color management

### PR DESCRIPTION
**Pull request by joanwolk**
### Ready
- use `ChromePicker` with alpha disabled
- handlers to set the nested state data for color and length of an existing color in the configuration
- add minimal style to `label` and `.chrome-picker` for legibility

### To-do
- add button to remove a color from the config
- add button to add a new color to the config
- style the picker to show/hide, allowing more compact display when not actively picking a color

### CODE REVIEW

File: src/Form.jsx [hunk 1]
```diff
@@ -46,40 +69,45 @@ const Form = ({ formData, setFormData }) => {
...
+      </div>
 
-      <IntegerInput
-        label="Number of rows:"
-        title="The number of rows displayed"
-        name="crows"
-        value={crows}
-        setValue={setValue}
-        validator={IntegerInput.validators.nonNegative}
+      <div>
```
Comment (alenia) [inline src/Form.jsx:93]: I’m a bit confused by this div (and also most of the divs wrapping checkbox inputs, but especially this one) These two things don’t seem like they should semantically be together

File: src/Form.jsx [hunk 2]
```diff
@@ -37,7 +42,25 @@ const Form = ({ formData, setFormData }) => {
...
+            /> 
+            <IntegerInput
+              label="Length:"
+              title="The number of stitches in this color segment"
+              name={`${index}`}
+              value={colorConfig[index].length}
+              setValue={setColorConfigLengthValue}
+              validator={IntegerInput.validators.nonNegative}
+            />
+            <br />
```
Comment (alenia) [inline src/Form.jsx:61]: Not a fan of using <br/> for layout unless you’re using it as a line break in text.

File: src/inputs/Integer.jsx [hunk 3]
```diff
@@ -7,7 +8,7 @@ const IntegerInput = ({ value, name, label, title, setValue, validator = () => t
     setValue(e.target.name, value);
   }
   return (
-      <div>
+      <React.Fragment>
```
Comment (alenia) [inline src/inputs/Integer.jsx:11]: oooh I actually style this in my open PR haha. I see what you mean now. I do think semantically it makes sense as a <div class=“input-group”/> or something like that

File: src/Form.jsx (hunk 2 above)
Comment (joanwolk) [reply src/Form.jsx:61]: I plan to remove this in later styling changes but it was good for immediate legibility

**Comment by alenia** [comment]
Whoops you didn't add your dependency in the package.json

File: src/Form.jsx (hunk 2 above)
Comment (joanwolk) [reply src/Form.jsx]: Removed `Fragment` and made it a `div` so I could remove this `br`

**Pull request merged**